updates for doc-2335 (#262)
harness/developer-hub: docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md

In **Monitored Service**, click **Click to autocreate a monitored service**.

> Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. For example, a Service named `todolist` and an Environment named `dev` will result in a monitored service with the name `todolist_dev`.

Reviewer: ```suggestion
Harness automatically creates a monitored service using a concatenation of the service and environment names. For example, a service named `todolist` and an environment named `dev` results in a monitored service with the name `todolist_dev`.
```

:::note

> The option to auto-create a monitored service is not available if you have configured either service or environment or both as runtime values. When you run the pipeline, Harness concatenates the service and environment values you enter in the runtime inputs screen and generates a monitored service name. If an existing monitored service with the same name exists, Harness assigns it to the pipeline. If no monitored service exists that matches the generated monitored service name, Harness skips the verification step. 

Reviewer: ```suggestion
The option to auto-create a monitored service is not available if you have configured a service, an environment, or both as runtime values. When you run the pipeline, Harness concatenates the service and environment values you enter in the runtime inputs screen and generates a monitored service name. If an existing monitored service with the same name exists, Harness assigns it to the pipeline. If no monitored service that matches the generated monitored service name exists, Harness skips the verification step. 
```

> For example, suppose you enter service as `todolist` and environment as `dev`. In that case, Harness generates the monitored service name `todolist_dev`, checks whether a monitored service with the name `todolist_dev` is available, and assigns it to the pipeline. If no monitored service is available with the name `todolist_dev`, Harness skips the verification step.

Reviewer: ```suggestion
For example, suppose you enter the service as `todolist` and the environment as `dev`. In that case, Harness generates the monitored service name `todolist_dev`, checks whether a monitored service with the name `todolist_dev` is available, and assigns it to the pipeline. If no monitored service is available with the name `todolist_dev`, Harness skips the verification step.
```
